UnitsNet/UnitDefinitions/DcElectricPotential.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "DcElectricPotential",
```

Comment:
We should probably mark the existing `ElectricPotential` as obsolete when introducing this.